docs/source/guides/optimizers.rst

```diff
@@ -0,0 +1,59 @@
+Optimizer
+~~~~~~~~~
+
+.. include:: ../imports.rst
+
+In order to get good values for parameters, we have to optimize them by the :class:`Optimizer` class.
```

Comment:
I think we should elaborate a bit on _good values_, e.g. that it should minimize the loss, give high prediction accuracy or something similar (but this also depends on the model I guess).